utils/set_test_modules

```diff
@@ -18,27 +17,42 @@ TOPDIR="$(dirname "${BASH_SOURCE[0]}")/.."
 
 pushd "${TOPDIR}" >/dev/null 2>&1 || die "Failed to change directory."
 
+SKIP_GIT_TEST=${SKIP_GIT_TEST:-"False"}
+if [ $# -gt 0 ] && [ "${1}" == "-I" ]
```

Comment:
Please convert this to getopts. It is more work, but also cleaner and simpler to extend.